cpu/esp_common/esp_common.config

```diff
@@ -0,0 +1,8 @@
+CONFIG_MODULE_RANDOM=y
+CONFIG_MODULE_PRNG_TINYMT32=y
```

Comment:
Where does this come from, I couldn't find it in any `Makefile*` for ESPs boards and the CPUs.

Reply:
ESP brings random by default. That module provides multiple backends. While in Makefile checking for HWRNG and default to it when possible is not easy to model, Kconfig provides this as a choice (see [here](https://github.com/RIOT-OS/RIOT/blob/44ebc38caca3ce24914a6b543fb34703a304e660/sys/random/Kconfig)). As Kconfig would default to HWRNG but Makefile does not, we are defaulting to a software implementation whenever it is used. See a discussion on this in https://github.com/RIOT-OS/RIOT/pull/17188

Reply:
Ok, it is only for compaibility of kconfig with Mafiles. Will this be changed once PR #16732 is merged, if it is merged? From my point of view it makes sense to use the HWRNG whenever it is possible.

Reply:
With #17188 merged now the default is `MODULE_PRNG_MUSL_LCG`, I'll adapt